docs/guide/README.md

```diff
@@ -27,7 +27,7 @@ The Web Component spec consist of four parts:
 
 Especially Custom Elements and Shadow DOM are great technologies for splitting your UI into small, specialized components or building UI component libraries. MoleculeJS focuses on these specifications and HTML Templates can be used by the renderer of your choice, like lit-html.
 
-To understand why using Molecule might be benificiary, let's look at Custom Elements without MoleculeJS, **Vanilla WebComponents**.
```

Comment:
*benificiary* is both misspelled —ben ***e*** ficiary— and misused; it's a noun referring to
> a natural person or other legal entity who receives money or other benefits from a benefactor

whereas, in this place, the sentence calls for an adjective meaning
> favorable or advantageous; resulting in good

— *beneficial*